### PR TITLE
adds change made by Bennett

### PR DESCRIPTION
**Pull request by bennettbuchanan**
Adds "THIS IS A CHANGE BY BENNETT"

### CODE REVIEW

File: screenplay.txt [hunk 1]
```diff
@@ -83,3 +83,5 @@ Integer turpis turpis, consequat id auctor at, elementum ut neque. Donec eleifen
 FADE OUT.
 
 THE END
+
+THIS IS A CHANGE BY BENNETT.
```
Comment (WilliamMcCann) [inline screenplay.txt]: I especially like this line.

Comment (WilliamMcCann) [inline screenplay.txt]: Line comment by William

Comment (WilliamMcCann) [inline screenplay.txt]: This is a line note too maybe?

**Comment by dalzuga** [comment]
William, I am forced to concur

**Pull request merged**